src/lib/list/list.md

```diff
@@ -42,6 +42,20 @@ element in an `<mat-list-item>`.
 </mat-nav-list>
 ```
 
+### Action lists
+
+Use `mat-action-list` tags for action lists (i.e. lists that have button tags).
```

Comment:
The phrasing should be:

> A `<mat-action-list>` element is used for action lists (i.e. lists that have button tags.

(This is based on the earlier section about navigation lists.)

Reply:
The navigation list is actually doing the same as what I have put in originally. And I feel "use xxx" is more clear than "xxx is used" here.